### PR TITLE
Change password playbook example, a new role manage_efm role and tablespace management in manage_dbserver

### PR DESCRIPTION
**Pull request by vibhorkumar123**
1. Updated manage_dbserver role to behave correctly in case pgpass doesn't exist and create attribute is not set to yes
2. added manage_efm role for managing efm parameters of EFM cluster
3. Adding playbook for changing repuser/efm password
4. Added table space management in mange_dbserver

### CODE REVIEW

File: roles/manage_dbserver/tasks/pgpass_line.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  set_fact:
+      search_line: "^{{ line_item.host|default('*') }}:{{ pg_port }}:{{ line_item.database|default('*') }}:{{ line_item.user }}"
+      pgpass_line: "{{ line_item.host|default('*') }}:{{ pg_port }}:{{ line_item.database|default('*') }}:{{ line_item.user }}:{{ line_item.password }}"
+  when:
+    - pg_pgpass_values|length > 0
+
+- name: delete entries in .pgpass
+  lineinfile:
+    path: "{{ pgpass_file | default('~/.pgpass') }}"
+    regexp: "{{ search_line|replace('*','\\*') }}"
```
Comment (jt-edb) [inline roles/manage_dbserver/tasks/pgpass_line.yml:21]: I'd recommend using `"{{ search_line | regex_escape() }}"`

File: roles/manage_efm/tasks/efm_cluster_set_params.yml [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+    line: "{{ line_item.name + '=' + line_item.value|string }}"
+    state: present
+  with_items: "{{ efm_parameters }}"
+  loop_control:
+   loop_var: line_item
+  when: efm_parameters|length > 0
+  become: yes
+  register: properties_changes
+
+- name: Update {{ pg_service }} to bind {{ efm_service }}
```
Comment (jt-edb) [inline roles/manage_efm/tasks/efm_cluster_set_params.yml:27]: Please use quotes for task names when using variables

File: roles/manage_efm/tasks/efm_cluster_set_params.yml [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+  loop:
+    - line: "BindsTo="
+      regexp: "^BindsTo=.*"
+      insertafter: "^\\[Unit\\]$"
+  when:
+    - efm_eager_failover|bool
+    - ansible_os_family == 'RedHat'
+  register: pg_service_changed
+
+- name: Force systemd to reread configs of {{ pg_service }} after changes
```
Comment (jt-edb) [inline roles/manage_efm/tasks/efm_cluster_set_params.yml:43]: Use quotes

File: roles/manage_efm/tasks/efm_cluster_set_params.yml [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+    - properties_changes.changed
+  become: true
+
+- name: Cleanup the efm lock file if exists
+  file:
+    path: "{{ edb_efm_lock_file }}"
+    state: absent
+  become: true
+
+- name: Update {{ pg_service }} to bind {{ efm_service }}
```
Comment (jt-edb) [inline roles/manage_efm/tasks/efm_cluster_set_params.yml:68]: Use quotes

File: roles/manage_dbserver/tasks/pgpass_line.yml (hunk 1 above)
Comment (vibhorkumar123) [reply roles/manage_dbserver/tasks/pgpass_line.yml]: Thank you for the tip! Fixed!

File: roles/manage_efm/tasks/efm_cluster_set_params.yml (hunk 2 above)
Comment (vibhorkumar123) [reply roles/manage_efm/tasks/efm_cluster_set_params.yml]: Fixed!

File: roles/manage_efm/tasks/efm_cluster_set_params.yml (hunk 4 above)
Comment (vibhorkumar123) [reply roles/manage_efm/tasks/efm_cluster_set_params.yml]: Fixed

**Review by jt-edb — APPROVED** [review]
LGTM

**Pull request merged**